photon-client/src/integTest/scala/com/linkedin/photon/ml/data/avro/AvroDataReaderIntegTest.scala

```diff
@@ -35,6 +35,17 @@ class AvroDataReaderIntegTest extends SparkTestUtils {
 
   import AvroDataReaderIntegTest._
 
+  @Test
+  def testFeedAvro(): Unit = sparkTest("feedTest") {
```

Comment:
Should choose another name for this test. 'Feed' is specific to LinkedIn. Furthermore, what the test is actually testing is reading Maps with multiple types or non-String types. Should also add some documentation explaining this.

Reply:
done.